server/data/premisesClient.test.ts

```diff
@@ -3,6 +3,7 @@ import nock from 'nock'
 import premisesFactory from '../testutils/factories/premises'
 import PremisesClient from './premisesClient'
 import config from '../config'
+import paths from '../paths/manage'
```

Comment:
This is an ace idea, but I feel like we should seperate the application paths from the API paths